packages/create-react-microservice/src/commands/default.spec.js

```diff
@@ -462,6 +462,47 @@ describe('new Command().safelyCreateNpmScopeArg()', () => {
       '@foo-bar-baz/'
     );
   });
+  it('should remove all special characters', async () => {
+    const specialChars = [
+      '~',
+      '!',
+      '@',
+      '#',
+      '$',
+      '%',
+      '^',
+      '&',
+      '*',
+      '(',
+      ')',
+      '_',
+      '|',
+      '+',
+      '=',
+      '?',
+      ';',
+      ':',
+      ',',
+      '.',
+      '<',
+      '>',
+      '{',
+      '}',
+      '[',
+      ']',
+      '\\',
+      '/',
+      "'",
+      '"',
+      '`',
+      '´'
+    ];
+    specialChars.forEach(character => {
+      expect(instance.safelyCreateNpmScopeArg(`${character}foo-bar`)).toBe(
```

Comment:
You could simplify this test a bit by using the `specialChars` with a `join('')`, that way you can remove the `forEach` iteration, the test is a bit leaner and you would also cover that the characters should be replaced globally and not just once - Aside from that little thing, awesome job! 💯